### PR TITLE
Adding mysql support

### LINKED ISSUE #497: Enhance the workflow management service to use RDS in place of SQLite
Schema should be the same between SQLite and RDS but cloud deployments should be configured to use RDS

### PR DESCRIPTION
**Pull request by wjcunningham7**
fixes #497

### CODE REVIEW

**Comment by wjcunningham7** [comment]
@FyzHsn this PR simply switches from sqlite to sqlalchemy, and I have tested that both sqlite and mysql (via RDS) work properly.  The next PR will focus on new schemas.

File: covalent_results/app/api/api_v0/endpoints/workflow.py [hunk 1]
```diff
@@ -121,37 +121,29 @@ async def _concurrent_download_and_serialize(semaphore, file_name, session):
...
+
     return rows
 
 
 def _get_result_from_db(dispatch_id: str, field: str) -> Optional[str]:
-    sql = f"SELECT {field} FROM results WHERE dispatch_id=?"
-    value = db.value(sql, key=dispatch_id)
-    if value:
-        (value,) = value
+    sql = f"SELECT {field} FROM workflow WHERE id = '{dispatch_id}'"
```
Comment (FyzHsn) [inline covalent_results/app/api/api_v0/endpoints/workflow.py:130]: I believe these queries at some point need to be rewritten using the ORM.

**Review by FyzHsn — APPROVED** [review]
Left one nitpick comment - otherwise looks good.

**Review by scottwn — APPROVED** [review]

**Comment by scottwn** [comment]
We will need to put an `@skip` on the SQL tests to get this merged. Similar to what we did for NATS https://github.com/AgnostiqHQ/covalent/commit/e6a83e8cc57deb7e42cdb39d6190ada791388422#diff-e30fc3dc6524b970e749f89f26059a8a30568f20b81515ef9b44cea69b71e8aeR36

**Review by AlejandroEsquivel — DISMISSED** [review]

**Review by AlejandroEsquivel — APPROVED** [review]

**Review by FyzHsn — APPROVED** [review]

**Pull request merged**